localnet/single/start.sh

```diff
@@ -4,11 +4,11 @@ command -v shellcheck >/dev/null && shellcheck "$0"
 
 # Please keep this in sync with the Ports overview in HACKING.md
 TENDERMINT_PORT_GUEST="26657"
-TENDERMINT_PORT_HOST="26457"
+TENDERMINT_PORT_HOST="26657"
 API_PORT_GUEST="1317"
-API_PORT_HOST="1316"
+API_PORT_HOST="1317"
 GRPC_PORT_GUEST="9090"
-GRPC_PORT_HOST="9089"
+GRPC_PORT_HOST="9090"
```

Comment:
If you don't wanna distinguish this port, I think we should remove it

Reply:
If this ports are not defined, the internal node port cannot be used outside of docker.

Reply:
I mean we can just define as GRPC_PORT, API_PORT without suffix

Reply:
The "-p" option of "docker run" need mapping both inside and outside port. So I think it is not bad because we can know what it is and change it easily later.